src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -8,16 +8,17 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Getter
-@Entity(name = "CARTS")
+@Entity
+@Table(name = "CARTS")
 public class Cart {
 
     @Id
     @NonNull
+    @GeneratedValue
     @Column(name = "ID", unique = true)
-    @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
It looks like relations @OneToOne should be EAGER according to this article: 

https://nullpointerexception.pl/trzy-rzeczy-ktore-powinienes-wiedziec-o-hibernate/

so I would leave it EAGER

Reply:
Ok, I've changed files as you written.